<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 66, 66);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(37, 39, 48);
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(34, 36, 44);}
QPushButton:hover {background-color: rgb(100, 100, 100) }
QPushButton:hover:pressed {background-color: rgb(150, 150, 150)}</string>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(37, 39, 48) ;
border-radius: 10px;
color: #FFF;
padding-left: 10px;
padding-right: 10px;
background-color: rgb(34, 36, 44);
</string>
    </property>
    <property name="placeholderText">
     <string>Широта</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px rgb(37, 39, 48) ;
border-style: solid;
border-radius: 10px;
color: #FFF;
padding-left: 10px;
padding-right: 10px;
background-color: rgb(34, 36, 44);
</string>
    </property>
    <property name="placeholderText">
     <string>Долгота</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>5</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 0, 0);
	border: 2px solid rgb(37, 39, 48);
	border-radius: 9px;
	color: #FFF;
	padding-left: 1px;
	padding-right: 1px;}
QPushButton:hover {background-color: rgb(170, 0, 0) }
QPushButton:hover:pressed {background-color: rgb(150, 150, 150)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px rgb(37, 39, 48) ;
border-style: solid;
border-radius: 10px;
color: #FFF;
padding-left: 10px;
padding-right: 10px;
background-color: rgb(34, 36, 44);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(34, 36, 44);
	border-radius: 10px;
	border: 1px solid rgb(37, 39, 48);
	padding: 10px;
	color: rgb(255, 255, 255);
	padding-left: 10px;
}
QComboBox:hover{
	border: 1px solid rgb(64, 71, 88);
	
}
QComboBox QAbstractItemView {
	color: rgb(75, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(50, 50, 50);
}</string>
    </property>
    <item>
     <property name="text">
      <string>Схема</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Спутник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Гибрид</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>5</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(66, 66, 66);
	border: 2px solid rgb(37, 39, 48);
	border-radius: 9px;
	color: #FFF;
	padding-left: 1px;
	padding-right: 1px;}
QPushButton:hover {background-color: rgb(100, 100, 100) }
QPushButton:hover:pressed {background-color: rgb(150, 150, 150)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
